addons/sale_project/models/project_project.py

```diff
@@ -300,6 +300,13 @@ def action_open_project_invoices(self):
             action['res_id'] = invoice_ids[0]
         return action
 
+    @api.model
+    def name_create(self, name):
+        res = super().name_create(name)
+        allow_billable = self.env.context.get('allow_billable')
+        if allow_billable:
+            self.browse(res[0])['allow_billable'] = True
+        return res
```

Comment:
No need, because we are passing allow_billable in the context. I don't know why we are doing this.